engine/api/src/main/scala/pl/touk/nussknacker/engine/api/defaults/ParameterDefaultValueDeterminer.scala

```diff
@@ -1,9 +0,0 @@
-package pl.touk.nussknacker.engine.api.defaults
```

Comment:
Moving this class to ui model in general is ok, but in this change we should also remove `ServiceLoader` usage in chain of determiners. Currently it is not usable because if someone want to load own determiner he/she need to add dependency to nk-ui module. It is not a good idea because it is huge module. AFAIK it wasn't used anywhere. Also programmer still has option to override this defaults via `NodeConfig`

Reply:
I've removed user strategies, done